notebooks/Designing a better simplifier.ipynb

```diff
@@ -2779,7 +2779,7 @@
    "source": [
     "As expected, this is a significant improvement in some cases. It is logically impossible that it could ever make things worse, but it's nice that it makes it better.\n",
     "\n",
-    "So far we've only looked at things where the interaction between elements was fairly light - the sum cases the values of other elements mattered a bit, but shrinking an integer could never enable other shrinks. Lets look at one where this is not the case: Where our condition is that we have at least 10 distinct elements."
+    "So far we've only looked at things where the interaction between elements was fairly light - the sum cases the values of other elements mattered a bit, but shrinking an integer could never enable other shrinks. Let's look at one where this is not the case: Where our condition is that we have at least 10 distinct elements."
```

Comment:
I think it's probably better to leave this notebook alone, as a historical / demo kind of thing.